doc/user_guide/dynamo0p3.rst

```diff
@@ -431,13 +431,18 @@ associated kernel metadata description and their precision:
 +--------------------------+------------------------+-----------+
 | R_SOLVER_OPERATOR_TYPE   | GH_OPERATOR            | R_SOLVER  |
 +--------------------------+------------------------+-----------+
+| R_TRAN_OPERATOR_TYPE     | GH_OPERATOR            | R_TRAN    |
++--------------------------+------------------------+-----------+
 | COLUMNWISE_OPERATOR_TYPE | GH_COLUMNWISE_OPERATOR | R_SOLVER  |
 +--------------------------+------------------------+-----------+
 
+Note, LMA and CMA operators can only have ``GH_REAL`` (:ref:`argument
```

Comment:
I must be missing something here: the table does contain LMA (`OPERATOR_TYPE`) and CMA (`COLUMNWISE_OPERATOR_TYPE`) entries. What am I missing (apart from coffee)?

Reply:
All other entries under Kernel Metadata column have argument type and data type (e.g. `GH_FIELD, GH_REAL`). This is omitted for operators. I agree, the note is confusing so I removed it and updated the operator entries with `GH_REAL` and re-aligned everything instead. I checked the build, the table is still within the text margins.

Reply:
Oh I see. Thanks for updating.